src/jdk.incubator.foreign/share/classes/jdk/incubator/foreign/NativeScope.java

```diff
@@ -181,6 +182,29 @@ default MemorySegment allocate(MemoryLayout layout, double value) {
         return addr;
     }
 
+    /**
+     * Allocate a block of memory in this native scope with given layout and initialize it with given address value.
+     * The address value might be narrowed according to the platform address size (see {@link MemoryLayouts#ADDRESS}).
+     * The segment returned by this method is associated with a segment which cannot be closed. Moreover, the returned
```

Comment:
This sentence seems strange.
```suggestion
     * The segment returned by this method cannot be closed. Moreover, the returned
```